docker-app/qfieldcloud/settings.py

```diff
@@ -220,21 +220,18 @@
 # Internationalization
 # https://docs.djangoproject.com/en/2.2/topics/i18n/
 
-LANGUAGE_CODE = "en"
+LANGUAGE_CODE = os.environ.get("QFIELDCLOUD_DEFAULT_LANGUAGE") or "en"
 
 TIME_ZONE = os.environ.get("QFIELDCLOUD_DEFAULT_TIME_ZONE") or "Europe/Zurich"
 
-USE_I18N = False
-
+USE_I18N = bool(os.environ.get("QFIELDCLOUD_USE_I18N")) or True
 
 USE_TZ = True
 
 
 LANGUAGES = [
-    ("de", "German"),
     ("en", "English"),
-    ("fr", "French"),
-    ("it", "Italian"),
+    ("es", "Spanish"),
 ]
```

Comment:
I guess in the future we want to generate this dynamically based on the available translated languages. Let's leave it hardcoded for now.